.github/workflows/ci.yml

```
@@ -619,3 +619,18 @@ jobs:
           done
 
           echo "::endgroup::"
+          
+  trigger-dependent-workflows:
+    runs-on: ubuntu-latest
+    needs:
+      - services-docker-image-publish
+      - docker-image-publish
+    steps:
+      - name: Send context to seperate pipeline
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.DISTRIBUTION_TOKEN }}
+          event-type: new_release
+          repository: RocketChat/Release.Distributions
+          client-payload: ${{ format('{"tag":"{0}"}', github.ref_name) }}
```

this is simpler and easier to read =)
```suggestion
          client-payload: '{"tag": "${{ github.ref_name }}"}'
```